flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/ExecutionVertex.java

```diff
@@ -283,15 +284,30 @@ public ArchivedExecution getPriorExecutionAttempt(int attemptNumber) {
         }
```

Comment:
OT: Can we remove the unused `LOG` field from this class?

Reply:
Will remove it.